user-group/survey01-email.html

```diff
@@ -1,114 +1,114 @@
 <body style="margin:0;padding:0">
- <div style="Margin:0;padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;min-width:100%;background-color:#EAECEC;">
-  <table style="border-spacing:0;width:100%;table-layout:fixed;">
-   <tbody>
-    <tr>
-     <td align="center" style="padding-top:30px;padding-bottom:0;padding-right:0;padding-left:0;vertical-align:top;">
+  <div style="Margin:0;padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;min-width:100%;background-color:#EAECEC;">
+    <table style="border-spacing:0;width:100%;table-layout:fixed;">
```

Comment:
We use 4 spaces per level of indentation in all other places. You should be able to set this as a preference in your editor and have it apply the correct formatting automatically